### ISSUE TITLE
Please disable, when having documents already open and working with two or more panes

### ISSUE THREAD

**Issue by twirley** [dy-sh/obsidian-remember-cursor-position#54]
I love this feature when I open documents.
But when working with two or more open files, clicking, dragging/dropping, this becomes horrible.

I click on a part on my open right window - a short moment later my courser jumps.
I click on the left again - my courser jumps.

I want my location remembered, when I open my document, not while I have it open working on it.
This is something Obsidian already does on its own, not interfering with clicking on certain parts :)

**Comment by benjaminwelch** [comment]
I would love to see an option in the settings for this as well. Otherwise, very useful!

**Comment by mbellezi** [comment]
I agree. Should have the option to disable when jumping between two opened files. Makes the use very difficult.

**Comment by YIRU69** [comment]
Agree

**Comment by SerafinDinges** [comment]
Agree. It's pretty annoying. Wonder if the trigger to go to scroll position could be set to first loading a document, instead of every focus.

**Comment by neilfranci** [comment]
Well, since the author doesn't update in the last 5 months. Someone needs to make a pull request(maybe not) or just fork it and make things update.

**Comment by dy-sh** [comment]
I can update the plugin if https://github.com/dy-sh/obsidian-remember-cursor-position/pull/56 solves the problem.

**Comment by neilfranci** [comment]
> I can update the plugin if https://github.com/dy-sh/obsidian-remember-cursor-position/pull/56 solves the problem.

nice, I'm using it rn. Rather than waiting for better solution. I just want "It's just work". It been holding for 5 months now. I don't think a better solution would come out in a minute  😂

**Comment by dy-sh** [comment]
updated the plugin with https://github.com/dy-sh/obsidian-remember-cursor-position/pull/56

**Comment by chrisspiegl** [comment]
Tested it in 1.0.9 and it appears to be working.

- Side by Side on the same document (and different scroll positions) ✅
- Side by Side with different documents ✅
- Opening the same document in multiple tabs ✅

Great work and so happy that this is now integrated. I had disabled this plugin becuase of this missing.
